docs/map.json

```diff
@@ -285,7 +285,7 @@
             },
             {
               "name": "How Caching Works",
-              "tags": ["cache-task-results"],
+              "tags": ["replay-task-results"],
```

Comment:
What do we do with the title though: "How Caching Works"

Do we leave "caching" here? And how would ppl relate the "replay" feature with the functionality?